DetectorDescription/DDCMS/src/DDFilteredView.cc

```diff
@@ -481,7 +481,8 @@ std::string_view DDFilteredView::get<string_view>(const string& key) const {
       int count = names.size();
       bool flag = false;
       for (int nit = level; count > 0 and nit > 0; --nit) {
-        if (!compareEqual(noNamespace(it_.back().GetNode(nit)->GetVolume()->GetName()), names[--count])) {
+        std::string_view name = noNamespace(it_.back().GetNode(nit)->GetVolume()->GetName());
```

Comment:
If you are going to force the construction of a `std::string` alway, why declare `name` as a `std::string_view` instead of as a `std::string`?

Reply:
good point. However, here it's done to make shorter the next line.